hibernate-reactive-core/src/main/java/org/hibernate/reactive/persister/collection/impl/ReactiveOneToManyPersister.java

```diff
@@ -55,4 +70,85 @@ protected ReactiveCollectionInitializer createSubselectInitializer(SubselectFetc
 	protected ReactiveCollectionInitializer getAppropriateInitializer(Serializable key, SharedSessionContractImplementor session) {
 		return (ReactiveCollectionInitializer) super.getAppropriateInitializer(key, session);
 	}
+
+	@Override
+	public CompletionStage<Void> recreateReactive(
+			PersistentCollection collection, Serializable id, SharedSessionContractImplementor session) {
+		return CompletionStages.voidFuture();
+	}
+
+	private ReactiveConnection getReactiveConnection(SharedSessionContractImplementor session) {
+		return ( (ReactiveConnectionSupplier) session ).getReactiveConnection();
+	}
+
+	@Override
+	public CompletionStage<Integer> removeReactive(Serializable id, SharedSessionContractImplementor session)
+			throws HibernateException {
+		ReactiveConnection reactiveConnection = getReactiveConnection( session );
+		if ( !isInverse && isRowDeleteEnabled() ) {
+			if ( LOG.isDebugEnabled() ) {
+				LOG.debugf(
+						"Deleting collection: %s",
+						MessageHelper.collectionInfoString( this, id, getFactory() )
+				);
+			}
+
+			// Remove all the old entries
+
+			int offset = 1;
+			final PreparedStatement st;
+			Expectation expectation = Expectations.appropriateExpectation( getDeleteAllCheckStyle() );
+			boolean callable = isDeleteAllCallable();
+
+			List<Object> params = new ArrayList<>();
+			params.add( id );
+			String sql = getSQLDeleteString();
+
+			return reactiveConnection.update( sql, params.toArray( new Object[0] ) );
+		}
+		return CompletionStages.completedFuture( -1 );
+	}
+
+	@Override
+	public CompletionStage<Void> reactiveDeleteRows(
+			PersistentCollection collection, Serializable id, SharedSessionContractImplementor session) {
+		if ( isInverse ) {
+			return CompletionStages.voidFuture();
+		}
+
+		if ( !isRowDeleteEnabled() ) {
+			return CompletionStages.voidFuture();
+		}
+
+		//TODO: See AbstractCollectionPersister#deleteRows
+		throw new UnsupportedOperationException();
+	}
+
+	@Override
+	public CompletionStage<Void> reactiveInsertRows(
+			PersistentCollection collection, Serializable id, SharedSessionContractImplementor session) {
+
+		if ( isInverse ) {
+			return CompletionStages.voidFuture();
+		}
+
+		if ( !isRowDeleteEnabled() ) {
+			return CompletionStages.voidFuture();
+		}
+
+		// FIXME: Not sure what to implement here for OneToMany use-case
+		// Maybe nothing due to the ReactiveCollectionUpdate/Remove/Recreate Action classes perform necessary operations
+		// on the collection (basically delete and replace rather than operating on individual rows
```

Comment:
I think you can remove these comments, You had to add these methods because they are in the interface and I suspect, I'm not sure, we are going to work on this when adding support for `@ManyToMany` or tests associations with maps

Reply:
I think this would be enough:
```
// TODO: See AbstractCollectionPersister#insertRows
```

Reply:
got it